### DISCUSSION TITLE
Model::create generate SQL error for non nullable fields

### DISCUSSION THREAD

**Discussion by goeil** [filamentphp/filament#3700] [Help]
Hello all filament users,
I cannot manage to get out of this trivial error:
I used a Resource class for my model (as described in the doc) with Filament Admin.

But when I click on 'create' button, the logic behind tries to create an model object in database, which is not possible, because there are some non-nullable fields described in my model.

I get a general error:
`1364 Field 'my_field' doen't have a default value`

Caused by Filament\Pages\Actions\CreateAction :

`            $record = $this->process(fn (array $data): Model => $model::create($data));`

What am I missing?
Any help welcomed!

Thanks a lot

**Comment by danharrin** [comment]
Make sure the database columns in your model are `$fillable`

**Reply by goeil** [reply]
Yes, they are indeed.

```
    protected $fillable = [
        'title',
        'slug',
        'user_id',

    ];

```
I think it doesn't change anything in the `$model::create($data)`
All I got is two values, `updated_at` and `created_at` in `$data`.

`insert into posts (updated_at, created_at) values (2022-08-27 11:10:45, 2022-08-27 11:10:45)`

**Comment by goeil** [comment]
The error occurs precisely when I click on 'create' button from list page.
When I access directly to `/admin/model/create`, everything is OK.

**Reply by danharrin** [reply]
Please open an issue with a reproduction URL so we can test out the bug
